contract/readme/USAGE.rst

```diff
@@ -23,9 +23,4 @@
 #. Contract templates can be created from the Configuration -> Contracts -> Contract Templates menu.
    They allow to define default journal, price list and lines when creating a contract.
    To use it, just select the template on the contract and fields will be filled automatically.
-
-* Contracts appear in portal to following users in every contract:
-
-.. image:: ../static/src/screenshots/portal-my.png
```

Comment:
Why removing these images?

Reply:
I think that images was meaningful when new feature was introduced. Moreover: images are not present un V. 13.0 modules (there are broken links). But, maybe, I'm wrong... If you don't agree I can take new snapshots and fix this.

Reply:
The idea is to showcase features in README, so they are relevant, but it's a "bug" that they weren't present in 13.0. @victoralmau can you please recover them?

Reply:
Ok. I'll recover them too.

Reply:
Done @pedrobaeza

Reply:
I have created these PR https://github.com/OCA/contract/pull/659 to add portal images in 13.0